index.html

```diff
@@ -349,6 +343,42 @@
     </section>
 
     <section>
+      <h2>Extensions to the <a>DocumentOrShadowRoot</a> Mixin</h2>
+
+      <dl title='partial interface DocumentOrShadowRoot' class='idl'>
+         <dt>readonly attribute Element? pointerLockElement</dt>
+         <dd>
+           <dfn title="pointerLockElement"></dfn>
+
+           <p>If the element, which is set as the target for mouse events while the pointer is
```

Comment:
I recommend you to use a retargeting algorithm as is so that we do not have a lot of specilized retargeting algorithms.
I am afraid that we will have a maintanace burden.

I think your definition works here because it is for DocumentOrShadowRoot, and it is for Pointer locked element.
If one of these assumption breaks, your definition will not work.
I have to review your definition carefully here...
 e.g. The case where the target's root is neither Document nor ShadowRoot. Have you considered this case??

I prefer having one algorithm, and we should refer it from everywhere if we can apply it, instead of customizing it in many places.